pyproject.toml

```diff
@@ -3,7 +3,9 @@ minversion = "6.0"
 addopts = "--ignore=openpilot/ --ignore=cereal/ --ignore=opendbc/ --ignore=panda/ --ignore=rednose_repo/ --ignore=tinygrad_repo/ --ignore=laika_repo/ -Werror --strict-config --strict-markers --durations=10"
 #cpp_files = "test_*" # uncomment when agnos has pytest-cpp and remove from CI
 python_files = "test_*.py"
-#timeout = "30"  # you get this long by default
+setup_timeout = 15
+execution_timeout = 30
+teardown_timeout = 15
```

Comment:
did you pull these from panda? the setup/teardown should be much faster here, i'd make it 2s